neo/test/iotest/test_neomatlabio.py

```diff
@@ -26,21 +29,30 @@ class TestNeoMatlabIO(BaseTestIO, unittest.TestCase):
     files_to_download = []
 
     def test_write_read_single_spike(self):
-        block1 = Block()
+        block1 = Block(name="test_neomatlabio")
         seg = Segment('segment1')
         spiketrain1 = SpikeTrain([1] * pq.s, t_stop=10 * pq.s, sampling_rate=1 * pq.Hz)
-        spiketrain1.annotate(yep='yop')
+        spiketrain1.annotate(yep='yop', yip=None)
         sig1 = AnalogSignal([4, 5, 6] * pq.A, sampling_period=1 * pq.ms)
         irrsig1 = IrregularlySampledSignal([0, 1, 2] * pq.ms, [4, 5, 6] * pq.A)
+        img_sequence_array = [[[column for column in range(2)] for _ in range(2)]
+                      for _ in range(2)]
+        image_sequence = ImageSequence(img_sequence_array, units='dimensionless',
+                               sampling_rate=1 * pq.Hz,
+                               spatial_scale=1 * pq.micrometer)
         block1.segments.append(seg)
         seg.spiketrains.append(spiketrain1)
         seg.analogsignals.append(sig1)
         seg.irregularlysampledsignals.append(irrsig1)
+        seg.imagesequences.append(image_sequence)
+
+        group1 = Group([spiketrain1, sig1])
+        block1.groups.append(group1)
 
         # write block
         filename = self.get_local_path('matlabiotestfile.mat')
         io1 = self.ioclass(filename)
-        io1.write_block(block1)
+        io1.write_block(block1, debug=False)
```

Comment:
So based on this you want debug for testing purposes?

Reply:
removed now, left in accidentally.